modules/meilisearch/examples_test.go

```diff
@@ -0,0 +1,44 @@
+package meilisearch_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/testcontainers/testcontainers-go"
```

Comment:
chore: please run `make lint` for this module so that imports are properly formatted.